app/views/print/niLetter.scala.html

```diff
@@ -47,9 +47,15 @@ <h3 class="heading-medium margin-top-small no-margin-bottom print-header__headin
     <div class="column-half document-address__sender">
       <div class="sender-details">
         <address>
-        @Html(Messages("label.nic_eo_hmrc_address"))
+          @Html(Messages("label.nic_eo_hmrc_address"))
         </address>
-        <p class="phone">@Messages("label.phone_text")<span class="phone-number">@Messages("label.phone_0300_200_3500")</span></p>
+        <p class="phone">@Messages("label.phone_text")<span class="phone-number">
+          @if(messages.lang.equals(Lang("en"))) {
```

Comment:
This seems pretty hacky to me, would it not be better to have a message called `label.contact_number` for example and have english and welsh numbers in the respective messages files?